examples/trials/benchmarking/automlbenchmark/README.md

```diff
@@ -0,0 +1,39 @@
+# Benchmark for Tuners
```

Comment:
All docs should be placed in `docs` folder.

Reply:
done